multimedia/VLC-X11/files/0005-configure-do-not-err-out-on-implicit-function-decl.patch

```diff
@@ -0,0 +1,31 @@
+From 8d870f73f214f220bf7415628f8f5a8fd8b012e2 Mon Sep 17 00:00:00 2001
+From: Sergey Fedorov <vital.had@gmail.com>
+Date: Mon, 15 Apr 2024 20:32:15 +0800
+Subject: [PATCH] configure: do not err out on implicit function decl
```

Comment:
Disabling the implicit function declaration error is not recommended. Allowing implicit function declarations may lead to incorrect program execution on Apple Silicon processors.

Reply:
Ok, I will restrict the patch to older systems and verify it builds on Sonoma without it.